catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/models/pausable.json

```diff
@@ -0,0 +1,20 @@
+{
+  "model": {
+    "kind": "model",
+    "name": "pausable",
+    "title": "Pausable",
+    "description": "Resume EIP to support resuming processing from last known offset.",
```

Comment:
I think the description is a copy from the resume EIP. It should be something about pausing

Reply:
Note to self: avoid copying class headers. 

Good catch, thanks!